src/System.CommandLine/Help/HelpBuilder.cs

```diff
@@ -522,7 +522,7 @@ protected string GetArgumentDescriptor(IArgument argument)
 
             string descriptor;
             var suggestions = argument.GetSuggestions().ToArray();
-            if (suggestions.Length > 0)
+            if (suggestions.Length > 0 && string.Join("|", suggestions).Length < 50)
```

Comment:
Any thoughts on how to do this without the magic number 50? Should it be based on the terminal width? And is the calculated table width thrown off by this?

@keboo?

Reply:
My thought would be to not have it automatically turn off at a fixed threshold. It is hard to say what that threshold should be (even with accounting for terminal width). I can imagine some program wanting lots of suggestions output, while others do not. I think the question is what should be the right API for including/excluding them. 
Perhaps a better solution would be to expand the `HelpBuilderExtension.Customize` method on options/arguments to provide a means of controlling this?

Reply:
Switched to a different approach, see new commit.